config/scenarios.form.yaml

```diff
@@ -118,4 +118,75 @@ main-mds-vanilla-with-grids:
     storage_units: ["li-ion battery", "iron-air battery"]
     H2_network: true
     gas_network: true
-    H2_retrofit: true
\ No newline at end of file
+    H2_retrofit: true
+
+main-calibration-run:
```

Comment:
I would expect more changes in term of techs : no dac, no DSM for EV, no V2G, no TES, no SMR CC